NeoSwiftTests/Base58Tests.swift

```diff
@@ -0,0 +1,28 @@
+//
```

Comment:
Let's merge Base58 tests and SHA-256 Tests into one file and name it something like NeoCrypto tests